docs/jwst/pathloss/reference_files.rst

```diff
@@ -52,3 +56,80 @@ as the behavious or each slice is considered identical.
 The NIRSPEC MSASPEC reference file has 2 sets of 4 extensions, one for the 1x1
 aperture size, and one for the 1x3 aperture size.  Currently there are no other
 aperture sizes.
+
+The NIRISS SOSS reference file has 1 extension in addition to the primary
+header unit.  It contains a 3-dimensional array of float32 correction values.
+The dimensions of the array are 1x2048x17.  The first dimension is a dummy to
+force the array dimensionality to be the same as the NIRSPEC reference file
+arrays.  The other 2 dimensions refer to the number of columns in the correction
+(the same as the number of columns in the science data) and the range of
+values for the Pupil Wheel position (PWCPOS).
+
+The headers associated with the PS extensions should contain the WCS
+information that describes what variables the correction depends on and
+how they relate to the dimensions of the correction array.
+
+For the NIRSPEC reference files (MSASPEC, FIXEDSLIT and IFU), the WCS keywords
+should look like this:
+
+======= ========== =========================================================================================
+Keyword Value      Comment
+======= ========== =========================================================================================
+CRPIX1  1.0        Reference pixel in fastest dimension
+CRVAL1  -0.5       Coordinate value at this reference pixel
+CDELT1  0.05       Change in coordinate value for unit change in index
+CTYPE1  'UNITLESS' Type of physical coordinate in this dimension
+======= ========== =========================================================================================
+
+This dimension expresses the decenter along the dispersion direction for a point source
+
+======= ========== =========================================================================================
+CRPIX2  1.0        Reference pixel in fastest dimension
+CRVAL2  -0.5       Coordinate value at this reference pixel
+CDELT2  0.05       Change in coordinate value for unit change in index
+CTYPE2  'UNITLESS' Type of physical coordinate in this dimension
+======= ========== =========================================================================================
+
+This dimension expresses the decenter along the direction perpendicular to the dispersion for a point source
+
+======= ========== =========================================================================================
+CRPIX3  1.0        Reference pixel in fastest dimension
+CRVAL3  6.0E-7     Coordinate value at this reference pixel
+CDELT3  2.35E-7    Change in coordinate value for unit change in index
+CTYPE3  'Meter'    Type of physical coordinate in this dimension (should be 'WAVELENGTH')
```

Comment:
Shouldn't a value like "meter" belong in a CUNITn keyword, rather than CTYPEn? Also, the comment suggests that this value should be "wavelength", but that of course doesn't adhere to the FITS WCS standard "4-3" format for CTYPEn. If it is a wavelength axis, the value should be "WAVE".

Reply:
The unit is "meter", not "Meter".  When a unit name is spelled out, it is never capitalized (except Angstrom, and only in FITS files).  Abbreviated units are capitalized iff it's a person's name (except L for liter).

Reply:
Thanks Phil.  I need to update these ref files at some point, I don't want to do it while we're getting an interim release put together.

Reply:
And for the case of fundamental quantities such meter, kilogram, and second the recommended abbreviations (for use in CUNITn keywords and units embedded in keyword comments) are just "m", "kg", and "s", respectively (not spelled out). So remember all these notes when the ref file updates are made in the future!

Reply:
Oh, if you're going to leave the ref file and documentation as is, I still think line 99 should be updated to say "should be 'WAVE'", as opposed to 'WAVELENGTH'. Not a big deal, but once the CTYPE3 value is updated, it should in fact have a value of "WAVE".